ctags/parsers/abaqus.c

```diff
@@ -17,6 +17,7 @@
 #include "parse.h"
 #include "read.h"
 #include "vstring.h"
+#include "routines.h"
```

Comment:
Yes but I didn't notice it at that point (abaqus isn't in uctags so I didn't see the diff). I'd prefer not having to manipulate the history because of this.